### DISCUSSION TITLE
人工智慧客製化自己的音樂！蘋果收購新創公司AI Music

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS3073-C/Python-Community-GS3073-C#399] [科技新知]
「根據外媒《MacRumors》報導，蘋果收購的新創公司AI Music開發「Infinite Music Engine」，利用人工智慧技術生成屬於每個人的音樂。此項技術甚至可以偵測心跳，創造出當下的音樂，也可以利用人工智慧寫出無版權音樂提供給創作者使用。」

我認為蘋果選擇收購實在是正確的選擇，因為音樂能發展的可能性很大，有無限的組合，期待有機會用到此AI技術生成自己的音樂

資料來源：https://www.google.com/amp/s/newtalk.tw/news/view/amp/2022-02-08/707060

**Comment by YuHengChen54** [comment]
這跟AI畫畫有異曲同工之妙欸，蠻酷的哈哈哈

**Comment by lx3211** [comment]
很好奇這樣創造出來的音樂會不會好聽呢
應該是參照很多和弦資料去訓練模型的？

**Comment by Chenszuhong** [comment]
感覺很有趣，但是可能會有些音樂很相似，畢竟是用心跳去設計

**Comment by xCk27x** [comment]
但感覺要跳脫統一的框架是一件難事
畢竟會紅的音樂都有自己的特色
不知AI製作出來的音樂能否有這樣的效果
是說我記得之前也有類似的報導
但結果是大部分的曲子都似是而非
沒有太多特色~

**Comment by limitx0** [comment]
如果有一些能免費開放的話
以後就不缺音樂素材了
